Spigot-Server-Patches/0601-Add-worldborder-events.patch

```diff
@@ -0,0 +1,293 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Jacob <jacobblair6789@gmail.com>
+Date: Mon, 16 Nov 2020 10:47:28 -0700
+Subject: [PATCH] Add worldborder events
+
+
+diff --git a/src/main/java/net/minecraft/server/WorldBorder.java b/src/main/java/net/minecraft/server/WorldBorder.java
+index f011869880fedae4b69e505491e8bdbc5f51dfba..c01b14e579773d6f32f8a00ce55f77c5c1ec5218 100644
+--- a/src/main/java/net/minecraft/server/WorldBorder.java
++++ b/src/main/java/net/minecraft/server/WorldBorder.java
+@@ -2,6 +2,14 @@ package net.minecraft.server;
+ 
+ import com.google.common.collect.Lists;
+ import com.mojang.serialization.DynamicLike;
++import io.papermc.paper.event.world.WorldBorderCenterChangeEvent;
++import io.papermc.paper.event.world.WorldBorderStartChangeEvent;
++import io.papermc.paper.event.world.StaticWorldBorderChangeEvent;
++import io.papermc.paper.event.world.WorldBorderFinishChangeEvent;
++import io.papermc.paper.event.world.MovingWorldBorderChangeEvent;
++import org.bukkit.Location;
++import org.bukkit.craftbukkit.CraftWorldBorder;
++
+ import java.util.Iterator;
+ import java.util.List;
+ 
+@@ -92,6 +100,10 @@ public class WorldBorder {
+     }
+ 
+     public void setCenter(double d0, double d1) {
```

Comment:
Isn't this also used by plugins? Why do the events take focus in the worldborder command if so?